projects/common/src/time/time-duration.ts

```diff
@@ -141,3 +151,8 @@ type ConvertibleTimeUnit =
   | TimeUnit.Minute
   | TimeUnit.Second
   | TimeUnit.Millisecond;
+
+export enum UnitStringType {
```

Comment:
I noticed this got switched back from const again? was const causing issues?

Reply:
yep. using const fails when the enum is used as a param value in a method.

Reply:
hmm - shouldn't be. I suspect that's a local caching artifact from the switch between const and non-const. Could you trying clearing your jest cache `npm run test -- --clear-cache` and then running with the const version?